docs/getting-started/vim.md

```diff
@@ -55,3 +50,53 @@ message from Coc:
 For suggestions on configuring Coc and possible key-bindings see its [example
 configuration
 documentation](https://github.com/neoclide/coc.nvim#example-vim-configuration).
+
+## Setup for Neovim using the built-in language server client
+
+The following instructions should enable Erlang language server integration
+using the built-in language server client in Neovim. The
+[nvim-lspconfig](https://github.com/neovim/nvim-lspconfig) plugin
+is used to configure, launch, and initialize the language server.
+
+### Installing nvim-lspconfig
+
+Install using, for example, [vim-plug](https://github.com/junegunn/vim-plug):
+
+```vim
+Plug 'neovim/nvim-lspconfig'
+```
+
+### Enable the language server
+
+Add the following setup call to your init.vim to make the language server attach
+to Erlang files:
+
+```vim
+lua << EOF
```

Comment:
Is this really supposed to say `lua`? :)

Reply:
Oops, no it is not :)